keras/layers/core.py

```diff
@@ -838,6 +838,7 @@ def __init__(self, units,
                  activity_regularizer=None,
                  kernel_constraint=None,
                  bias_constraint=None,
+                 sparse_weight=False,
```

Comment:
Add explanation in doc string, note only used for mxnet

Reply:
done